interface/wc_decision_icons.gfx

```diff
@@ -320,4 +320,8 @@ spriteTypes = {
 		name = "GFX_form_korkron"
 		texturefile = "gfx\\interface\\decision_icons\\form_korkron.dds"
 	}
+	spriteType = {
+		name = "GFX_forge_hammer_of_the_twilight"
+		texturefile = "gfx\\interface\\decision_icons\\forge_hammer_of_the_twilight.dds"
+	}
```

Comment:
Purge with fire!
P.S. ...since the decision is gone.

Reply:
sry, overlooked this one... just checked the text files sej changed... moved to unused files